hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/utils/TestUtils.java

```diff
@@ -118,6 +119,10 @@ public static StreamReadMonitoringFunction getMonitorFunc(Configuration conf) {
     return new StreamReadMonitoringFunction(conf, new Path(basePath), TestConfigurations.ROW_TYPE, 1024 * 1024L, null);
   }
 
+  public static MockStreamingRuntimeContext getMockRuntimeContext() {
+    return new org.apache.hudi.sink.utils.MockStreamingRuntimeContext(false, 4, 0);
```

Comment:
Are hard-coded values passed to construct `MockStreamingRuntimeContext` sufficient for all tests?

Reply:
yeah